### DISCUSSION TITLE
Want to contribute

### DISCUSSION THREAD

**Discussion by ravichigurupalli** [dotnet-architecture/eShopOnContainers#2091] [General]
Hello, This Is Ravi.
I am a software senior principal engineer form Austin, USA, I like the idea. I want to contribute in this project.         
Thanks.

**Comment by behdad088** [comment]
Hi @ravichigurupalli 
You can always contribute here, this project  accept pull requests from the community. When doing it, you can fork your own copy, please do it in the DEV branch which is the consolidated work-in-progress branch, contains the latest code, and is the branch actively developed.
Note that all PRs must be against the dev branch to be considered. This branch is developed using .NET 6.

You can begin with [README.md](https://github.com/dotnet-architecture/eShopOnContainers#readme) and for more information, you can follow this wiki page: https://github.com/dotnet-architecture/eShopOnContainers/wiki. I hope it helps you to start.
